packages/shared/src/messages/MessageProxy.ts

```diff
@@ -200,14 +205,14 @@ export class RpcBrowser {
       args: args,
     } as IMessageRequest);
 
-    // Add a timeout of 10 seconds for each call. However, if there is any "special" call that should not have a timeout, we can add a check here.
+    // Add a timeout of 5 seconds for each call. However, if there is any "special" call that should not have a timeout, we can add a check here.
     if (!specialChannels.includes(channel)) {
       setTimeout(() => {
         const { reject } = this.promises.get(requestId) ?? {};
         if (!reject) return;
         reject(new Error('Timeout'));
         this.promises.delete(requestId);
-      }, 10000); // 10 seconds
+      }, 5000); // 5 seconds
```

Comment:
```suggestion
      }, 5_000); // 5 seconds
```

nit

Reply:
There is one more feature in the AI Lab message proxy (multi-channel support) that isn't here. I plan to update to that in #1265 and will do this there.